docs/sources/administration/organization-preferences/index.md

```diff
@@ -237,3 +237,36 @@ You can choose your own personal home dashboard. This setting overrides all home
 1. On the left menu, hover your cursor over your avatar and then click **Preferences**.
 1. In the **Home Dashboard** field, select the dashboard that you want to use for your home dashboard. Options include all starred dashboards.
 1. Click **Save**.
+
+## Change Grafana language
```

Comment:
We want to avoid stacked headings, which is a heading immediately followed by another heading. I recommend that you add a short intro sentence below this heading. For example: "The following sections describe how to configure language for users, organizations, and teams."